src/components/screens/Archive/ArchiveNotes.jsx

```diff
@@ -0,0 +1,66 @@
+import axios from "axios";
+import React from "react";
+import { useArchiveContext } from "../../../global-context/archive-context";
+import { useGlobalVarContext } from "../../../global-context/global-variables";
+import { useNotesContext } from "../../../global-context/notes-context";
+import { useTrashContext } from "../../../global-context/trash-context";
+import {
+  postArchivesService,
+  transferFromArchiveToTrashService,
+} from "../../../services/archive-services";
+
+const ArchiveNotes = ({ archiveObj, setEditObj }) => {
+  const { _id, title, content, isPinned, colorPref, stylePref } = archiveObj;
+  const { token } = useGlobalVarContext();
+  const { dispatch: archiveDispatch } = useArchiveContext();
+  const { dispatch: notesDispatch } = useNotesContext();
+  const { dispatch: trashDispatch } = useTrashContext();
```

Comment:
Liked the way you are implementing the variable name!!

Reply:
Thank you